kuma/settings/common.py

```diff
@@ -764,6 +763,15 @@ def pipeline_one_scss(slug, **kwargs):
         ),
         'output_filename': 'build/styles/home.css',
     },
+    'print': {
+        'source_filenames': (
+            'styles/minimalist/print.scss',
```

Comment:
Oh, no. I have a little side project called `mdn-minimalist` and this will become(if I get to suggest/decide) the new CSS that is used for MDN Web Docs. I am therefore slowly starting to mimic the directory structure inside Kuma ;p